thunder/torch/__init__.py

```diff
@@ -5636,7 +5637,13 @@ def check_overlap_ops():
 
 
 # ref: https://pytorch.org/docs/stable/tensor_view.html
-_syms_returning_runtime_dependently_views: set[Symbol] = {reshape, contiguous, to, flatten}
+_syms_returning_runtime_dependently_views: set[Symbol] = {
```

Comment:
`_syms_that_may_return_views` might be a clearer name

A comment explaining what these operations are would be helpful, as well as an explanation of how this set works